apps/console/src/features/groups/components/wizard/group-basics.tsx

```diff
@@ -174,28 +152,28 @@ export const GroupBasics: FunctionComponent<GroupBasicProps> = (props: GroupBasi
      * The following function fetch the user store list and set it to the state.
      */
     const getUserStores = () => {
-        const storeOptions = [
+        const storeOptions: StoreOption[] = [
```

Comment:
`DropdownItemProps[]` should be able to use here, instead of a custom type `StoreOption[]`

Reply:
Fixed with b076d597fcdb4c9acc769a503b0fb20262ba0309